engine/src/main/java/org/terasology/rendering/dag/nodes/CopyImageToScreenNode.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2016 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.rendering.dag.nodes;
+
+import org.terasology.assets.ResourceUrn;
+import org.terasology.rendering.dag.ConditionDependentNode;
+import org.terasology.config.Config;
+import org.terasology.monitoring.PerformanceMonitor;
+import org.terasology.registry.In;
+import static org.lwjgl.opengl.GL11.*;
+
+import org.terasology.rendering.dag.stateChanges.BindFBO;
+import org.terasology.rendering.dag.stateChanges.EnableMaterial;
+import org.terasology.rendering.opengl.fbms.DisplayResolutionDependentFBOs;
+import org.terasology.rendering.world.WorldRenderer;
+import static org.terasology.rendering.opengl.OpenGLUtils.renderFullscreenQuad;
+import static org.terasology.rendering.opengl.DefaultDynamicFBOs.FINAL;
+import static org.terasology.rendering.world.WorldRenderer.RenderingStage.LEFT_EYE;
+import static org.terasology.rendering.world.WorldRenderer.RenderingStage.MONO;
+
+public class CopyImageToScreenNode extends ConditionDependentNode {
+
+    @In
+    private WorldRenderer worldRenderer;
+
+    @In
+    private Config config;
+
+    @In
+    private DisplayResolutionDependentFBOs displayResolutionDependentFBOs;
+
+    private static final ResourceUrn DEFAULT_FRAME_BUFFER_URN = new ResourceUrn("engine:display");
+
+    @Override
+    public void initialise() {
+        requiresCondition(() -> worldRenderer.getCurrentRenderStage() == MONO || worldRenderer.getCurrentRenderStage() == LEFT_EYE);
+        addDesiredStateChange(new BindFBO(DEFAULT_FRAME_BUFFER_URN,displayResolutionDependentFBOs));
+        addDesiredStateChange(new EnableMaterial("engine:prog.defaultTextured"));
+    }
+
+    @Override
+    public void process() {
+        PerformanceMonitor.startActivity("rendering/copyImageToScreen");
+        FINAL.bindTexture();
+        glClear(GL_COLOR_BUFFER_BIT | GL_DEPTH_BUFFER_BIT);
```

Comment:
Most nodes currently have a glClear call but it is my intention to verify if they are strictly necessary or if they could clear perhaps only the color or only the depth buffer. My reasoning is that in some cases all the pixels are getting rewritten anyway and clearing the output buffers to a default value is unnecessary.

I will be able to verify this on all standard nodes, but would you mind checking what impact would have removing the call or perhaps only clearing one of the two buffers in the VR-related nodes? Right now I'm not even sure the default frame buffer has a depth buffer.